src/Core/Style.js

```diff
@@ -335,7 +337,7 @@ function _addIcon(icon, domElement, opt) {
  * polygons will be extruded by the specified amount
  *
  * @property {Object} [stroke] - Lines and polygons edges.
- * @property {String|Function|THREE.Color} [stroke.color] The color of the line. Can be any [valid
+ * @property {String|Function|THREE.Color} [stroke.color] The color of the line. Can be any AAA [valid
```

Comment:
Maybe some residuals of a vim shortcut ?

Reply:
Yeah, you're right!